### PR TITLE
Update qbods.py

### PR DESCRIPTION
**Pull request by kathryn-ods**
Adding q822 - function to count how many companies have been dissolved within one year of founding. Function developed as part of red flags work.

### CODE REVIEW

**Comment by kathryn-ods** [comment]
@kd-ods ok to merge this?

**Review by kd-ods — COMMENTED** [review]
If there are no foundingDates in the original dataframe, then it falls over. Maybe worth handling that more carefully?

File: qbods.py [hunk 1]
```diff
@@ -1551,6 +1551,39 @@ def q821(entityStatement):
...
+    Returns: Dataframe of number of companies, grouped by year of founding. Barchart showing these counts. 
+    '''
+    
+    entSD = entityStatement.copy(deep=True)
+
+    entSD = entSD.dropna(subset=['dissolutionDate'])
+    entSD['name'] = entSD['name'].str.lower()
+    entSD = entSD.drop_duplicates(subset=['foundingDate','dissolutionDate','name'])
+
+    entSD['foundingDate'] = pd.to_datetime(entSD['foundingDate'],format='%Y/%m/%d')
```
[the remark this answers is not in the document]
Comment (kathryn-ods) [reply qbods.py:1570]: Yes it should be - it still seems to work with this format but it shouldn't!

**Comment by kathryn-ods** [comment]
@kd-ods have made those edits :)

**Review by kd-ods — APPROVED** [review]
Yup - this is working as expected. Thanks, Kathryn!

**Pull request merged**